### DISCUSSION TITLE
Update design creation.

### DISCUSSION THREAD

**Discussion by Wayder228** [tyron12233/CodeAssist#471] [Ideas]
Make it so that the design of the application can be created without code, it will be good. It already exists, but for example, I cannot drag the same buttons from place to place, change the color of the button, and so on.

**Comment by jefeter15** [comment]
Eu percebi que o CodeAssist precisa melhorar a criação de design eu preciso muito mesmo dessa ferramenta mais os meus projetos quase não avança por causa das limitações de criação então séria bom dar uma atualizada na ferramenta.

Eu agradeço por ler o meu comentário.
